src/sqlfluff/core/dialects/dialect_ansi.py

```diff
@@ -644,17 +647,28 @@ class OverClauseSegment(BaseSegment):
     match_grammar = Sequence(
         "OVER",
         Bracketed(
-            Sequence(
-                Ref("PartitionClauseSegment", optional=True),
-                Ref("OrderByClauseSegment", optional=True),
-                Ref("FrameClauseSegment", optional=True),
-                optional=True,
-                ephemeral_name="OverClauseContent",
+            OneOf(
+                Ref("WindowSpecificationSegment"),
```

Comment:
Here, I extracted the `Sequence()` above into `WindowSpecificationSegment`. This allows reusing the same window specification grammar for use in two places:
* Directly within an `OVER` clause (already supported)
* As part of a named `WINDOW` clause (new in this PR)